mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py

```diff
@@ -119,26 +133,49 @@ def native_query(self, query: str) -> Response:
 
     def query(self, query: ASTNode) -> Response:
         """
-        Retrieve the data from the SQL statement with eliminated rows that dont satisfy the WHERE condition
+        Executes a SQL query represented by an ASTNode and retrieves the data.
+
+        Args:
+            query (ASTNode): An ASTNode representing the SQL query to be executed.
+
+        Returns:
+            Response: The response from the `native_query` method, containing the result of the SQL query execution.
         """
         renderer = SqlalchemyRender(BigQueryDialect)
         query_str = renderer.get_string(query, with_failback=True)
         return self.native_query(query_str)
 
     def get_tables(self) -> Response:
         """
-        Get a list with all of the tabels in BigQuery
+        Retrieves a list of all non-system tables and views in the configured dataset of the BigQuery warehouse.
+
+        Returns:
+            Response: A response object containing the list of tables and views, formatted as per the `Response` class.
+        """
+        query = f"""
+            SELECT table_name, table_schema, table_type,
```

Comment:
This shouldn't be an issue but better to remove the comma after `table_type`

Reply:
Done.